README.md

```diff
@@ -166,8 +220,8 @@ make start
 
 ```shell
 cd frontend
-yarn
-yarn dev
+npm install
```

Comment:
I can revert this change if folks want to stick with yarn as opposed to npm. But the `npm install` step updates the `yarn.lock` file.